src/components/LayerUI.tsx

```diff
@@ -186,9 +182,44 @@ const LayerUI = ({
     );
   };
 
-  const [isMenuOpen, setIsMenuOpen] = useAtom(isMenuOpenAtom);
-  const menuRef = useOutsideClickHook(() => setIsMenuOpen(false));
+  const renderMenu = () => {
+    return (
+      childrenComponents.Menu || (
+        <Menu>
+          <Menu.DefaultItems.LoadScene />
+          <Menu.DefaultItems.SaveToActiveFile />
+          {UIOptions.canvasActions.export && <Menu.DefaultItems.Export />}
+          {UIOptions.canvasActions.saveAsImage && (
+            <Menu.DefaultItems.SaveAsImage />
+          )}
+          {onCollabButtonClick && (
+            <Menu.Item
+              dataTestId="collab-button"
+              icon={UsersIcon}
+              className={clsx({
+                "active-collab": isCollaborating,
+              })}
+              onClick={onCollabButtonClick}
+            >
+              {t("labels.liveCollaboration")}
+            </Menu.Item>
+          )}
```

Comment:
Since we're using this in host, and it may be useful to other host apps as well, let's factor this into a `DefaultItems.StartCollaboration` (better name?).

Reply:
"LiveCollaboration" since that's how we are referring to it ?

Reply:
yep